proto/connectrpc/conformance/v1/client_compat.proto

```diff
@@ -162,3 +152,15 @@ message ClientResponseResult {
 message ClientErrorResult {
   string message = 1;
 }
+
+message WireDetails {
+  // The HTTP status code of the response.
+  int32 actual_status_code = 1;
+  // When processing an error from a Connect server, this should contain
+  // the actual JSON received on the wire.
+  google.protobuf.Struct connect_error_raw = 2;
+  // Any HTTP trailers observed after the response body. These do NOT
+  // include trailers that conveyed via the body, as done in the gRPC-Web
+  // and Connect streaming protocols.
+  repeated Header actual_http_trailers = 3;
```

Comment:
Consider moving the `feedback` field from `ClientCompatResponse` to here. Then you could also put any error message from trying to compute wire details into that field.

Reply:
In that case then, we should probably rename `WireDetails` to something else since it no longer is just details about the wire. Unless we consider the feedback a 'wire' detail.  But, maybe something to reflect it's a reference-client only field i.e. `ReferenceClientMetadata` or something.

Reply:
I think the feedback would mostly be about wire details, so it would be fine to keep the name.

I don't actually have any examples of how we'd use and added it just for symmetry with the server. The server currently just uses it to complain if the incoming request doesn't have the expected properties (like HTTP version, protocol, codec, compression, etc). Technically, those are really wire details. Semantics are totally handled via the normal assertions in the test runner that compare the expected and actual responses.